interview-questions.md

```diff
@@ -8,62 +8,63 @@ Then, research the question to expand on your answer. Even if you feel you have
 
```

Comment:
For your interview questions, please add more of your experience with the answers. Tell your thoughts as well as what you researched. These interview questions are opportunities to practice sharing your thoughts with an interviewer/future clients.